Software/real_time/ROS_RoboBuggy/src/robobuggy/launch/transistor.launch

```diff
@@ -1,21 +1,24 @@
 <launch>
-    <node name="Transistor_Localizer" pkg="robobuggy" type="Transistor_Localizer" required="true">
```

Comment:
Any reason why these were changed? These are all essential nodes for running the system and should remain required

Reply:
My bad, I removed this to test it, will change.